tests/acceptance/01_vars/02_functions/string_replace.cf

```diff
@@ -0,0 +1,43 @@
+
+#######################################################
+#
+# Test string_replace function
+#
+#######################################################
+
+body common control
+{
+      inputs => { "../../default.cf.sub" };
+      bundlesequence  => { default("$(this.promise_filename)") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent test
+{
+  vars:
+      "test" string => "abcdefghij";
+
+      # replace the empty string with empty string, globally
+      "void" string => string_replace($(test), "", "");
+      # replace the empty string with 'x', globally (matches on character boundaries)
```

Comment:
What does *globally* mean here? I guess it's just a copy-paste from some other test?

Reply:
Yep, it's from regex_replace. Globally means that it adds the replacement character between every character. The reason I included it in the tests is if we want that functionality.

Reply:
But there's no such argument for the function that would enable/disable this, right? So I think it's quite confusing in the comment.